acslib/ccure/search.py

```diff
@@ -160,9 +160,9 @@ def __init__(
         self.outer_bool = f" {outer_bool.value} "
         self.inner_bool = f" {inner_bool.value} "
         self.term_operator = term_operator.value
-        self.display_properties = display_properties
-        if display_properties is None:
-            self.display_properties = ["Name"]
+        self.display_properties = ["Name"]
+        if display_properties is not None:
```

Comment:
It is not necessary to explicitly say `is not None` if `display_properties` is [None, "", {}, []] it will evaluate as False in the if condition.  What you have is  perfectly fine python but I just wonder why the choice?

Reply:
I touched on that here: https://github.com/ncstate-sat/acslib/pull/4#discussion_r1528658120